### PR TITLE
Bug 1850785 - Allow for Marionette failures to be filed as single tra…

### PR DESCRIPTION
**Pull request by jmaher**
…cking bugs. r=aryx

### CODE REVIEW

**Review by Archaeopteryx — CHANGES_REQUESTED** [review]

**Comment by jmaher** [comment]
thanks for the drive by; this patch is probably 50% complete, I hope to have it done this week and will request a review when it is ready.  I will support Marionette as well as some talos errors

File: treeherder/model/error_summary.py [hunk 1]
```diff
@@ -234,6 +234,8 @@ def get_error_search_term_and_path(error_line):
             # For reftests, remove the reference path from the tokens as this is
             # not very unique
             test_name_or_path = REFTEST_RE.sub("", test_name_or_path).replace("\\", "/")
+            # split marionette paths to only include the filename
+            test_name_or_path = "%s.py" % test_name_or_path.split(".py ")[0]
```
[the remark this answers is not in the document]
Comment (jmaher) [reply treeherder/model/error_summary.py:238]: I need to revisit this logic, thanks for pointing out the need to check if .py exists.  Ideally I can find a way to determine if this is marionette based on other path or test line specifics.

**Comment by jmaher** [comment]
these changes will allow for bugfiler to work properly for talos crashes and marionette regular failures.

In addition there are some changes to the bug suggestions (backend) which will find a marionette test line and strip it to match the test path, not the full failure path.

**Comment by Archaeopteryx** [comment]
This landed with #7810. Can you close it?

**Pull request closed without merging**